src/elements/content-uploader/DroppableContent.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+
+import ItemList from './ItemList';
+import UploadState from './UploadState';
+
+import makeDroppable from '../common/droppable';
+import type { UploadFile, UploadFileWithAPIOptions, UploadItem } from '../../common/types/upload';
+import type { DOMStringList, View } from '../../common/types/core';
+
+import './DroppableContent.scss';
+
+export interface DroppableContentProps {
+    addDataTransferItemsToUploadQueue: (droppedItems: DataTransfer, itemUpdateCallback?: Function) => void;
+    addFiles: (
+        files?: Array<UploadFileWithAPIOptions | UploadFile>,
+        itemUpdateCallback?: Function,
+        isRelativePathIgnored?: boolean,
+    ) => void;
+    allowedTypes: Array<string>;
```

Comment:
added missing prop `allowedTypes` since it is being passed from contentUploader file
https://github.com/box/box-ui-elements/blob/ba299b1dcee18d3197e5f7df0f15823070a071cb/src/elements/content-uploader/ContentUploader.js#L1292-L1301